bin/deploy-cloudstate.sh

```diff
@@ -39,20 +39,20 @@ else
   done
 fi
 
-
-# Use a custom, non-existant docker hub username so that we don't pull down the wrong thing accidentally.
-# Build the operator, required proxies, and compile the K8s descriptors.
 [ -f docker-env.sh ] && source docker-env.sh
-sbt -Ddocker.username=cloudstatedev -Ddocker.tag=dev -Duse.native.builds=$native operator/docker:publishLocal operator/compileK8sDescriptors "${build_commands[@]}"
 
-# Deploy Cloudstate operator
-kubectl create namespace cloudstate
-kubectl apply -n cloudstate -f operator/cloudstate-dev.yaml
-echo
+if [ -n "$build_commands" ]
+then
+  sbt "$build_commands"
+fi
+
+kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v0.16.1/cert-manager.yaml
+make -C cloudstate-operator deploy
 echo "Waiting for operator deployment to be ready..."
-if ! kubectl wait --for=condition=available --timeout=2m -n cloudstate deployment/cloudstate-operator
+if ! kubectl wait --for=condition=available --timeout=2m -n cloudstate-operator-system deployment/cloudstate-operator-controller-manager
```

Comment:
cloudstate-operator-controller-manager … -provider-factory-bean :)

Reply:
Why change the namespace from cloudstate to cloudstate-operator-system? Much more verbosity.

Reply:
Yeah, `cloudstate-operator-system` doesn't make sense. I'll rename to `cloudstate-system`, the `-system` suffix is a convention for operators that I think is a good idea to follow.